<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Dialog{border-image: url(:/over.jpg);}</string>
  </property>
  <widget class="QLineEdit" name="user_lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(255, 255, 255); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="enter_pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(255, 132, 139); border-radius: 3px; color: rgb(255, 255, 255); } 
QPushButton:hover { background-color:rgb(85, 255, 127); }

</string>
   </property>
   <property name="text">
    <string>enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(255, 132, 139); border-radius: 3px; color: rgb(255, 255, 255); } 
QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
  <zorder>quit_pushButton</zorder>
  <zorder>enter_pushButton</zorder>
  <zorder>password_lineEdit</zorder>
  <zorder>user_lineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
